src/unity/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+# cmake file for ripple-libpp
+
+# Include this library in your own CMake project by including the following somewhere
+# below your project definition (`project(your_application)`)
+#
+# add_subdirectory(extras/ripple-libpp/src/unity)
+#
+# Customize the path as needed
+
+cmake_minimum_required(VERSION 3.1.0)
+
+list(APPEND CMAKE_MODULE_PATH
+  "${CMAKE_CURRENT_SOURCE_DIR}/../../extras/rippled/Builds/CMake")
+include(CMakeFuncs)
+
+include_directories(
+  ../../extras/rippled/src
+  ../../extras/rippled/src/beast/include
+  ../../extras/rippled/src/beast/extras)
+#  ../../extras/ripple-libpp/extras/rippled/src/srcp256k1
+#  ../../extras/ripple-libpp/extras/rippled/src/ed25519-donna
+
+determine_build_type()
+
+use_boost(
+  regex
+  system
+  thread)
+
+use_openssl(1.0.1)
```

Comment:
It's a little sneakier than that. We start with 1.0.2 in `src/test/CMakeLists.txt`, but then we call `parse_target()`, which will override it to 1.0.1 if one if it has `ci` as a component, because that's the latest version we can get on the current Travis environment. However, this file does not call `parse_target()`, because I didn't want the dependency, so I just leave it as 1.0.1 here.